apps/bsatool/bsatool.cpp

```diff
@@ -14,54 +14,51 @@
 namespace bpo = boost::program_options;
 namespace bfs = boost::filesystem;
 
-struct Arguments
-{
+struct Arguments {
     std::string mode;
     std::string filename;
     std::string extractfile;
     std::string outdir;
 
-    bool longformat;
-    bool fullpath;
+    bool longformat{};
+    bool fullpath{};
 };
 
 void replaceAll(std::string& str, const std::string& needle, const std::string& substitute)
 {
     size_t pos = str.find(needle);
-    while(pos != std::string::npos)
+    while (pos != std::string::npos)
     {
         str.replace(pos, needle.size(), substitute);
         pos = str.find(needle);
     }
 }
 
-bool parseOptions (int argc, char** argv, Arguments &info)
+bool parseOptions(int argc, char** argv, Arguments& info)
 {
     bpo::options_description desc("Inspect and extract files from Bethesda BSA archives\n\n"
-            "Usages:\n"
-            "  bsatool list [-l] archivefile\n"
-            "      List the files presents in the input archive.\n\n"
-            "  bsatool extract [-f] archivefile [file_to_extract] [output_directory]\n"
-            "      Extract a file from the input archive.\n\n"
-            "  bsatool extractall archivefile [output_directory]\n"
-            "      Extract all files from the input archive.\n\n"
-            "Allowed options");
+                                  "Usages:\n"
```

Comment:
Is that an improvement? I think we are not entirely consistent with indenting in cases like this. But from what I remember the bulk of the code is using the style present before this change. So this would decrease consistency overall.

Reply:
We had a talk about this with the team, the idea is to create a .clangformat that everyone can use so that we're all on the same page consistency. The point of my PR was not style, but it seems to have gone in that direction due my K&R leanings.  Moving into Allman style.